scripts/dasharo-deploy

```diff
@@ -483,238 +931,73 @@ install() {
   sleep 0.5
   echo "Rebooting"
   sleep 1
-  if [ "$NEED_EC_RESET" = "true" ]; then
+  if [ "$NEED_EC_RESET" == "true" ]; then
     it5570_shutdown
   else
     send_dts_logs
```

Comment:
Not your change but it should probably go before if/else to always send logs?

Reply:
Thanks, very important catch! Fixed [here](https://github.com/Dasharo/dts-scripts/compare/a29856f198488767178cbe8e15853f96def1fb8b..ed6c6089caa0e541f15150b15a79a7e26c6ae365).